### ISSUE TITLE
Zip - Windows only?

### ISSUE THREAD

**Issue by ghost** [Shevchik/AutoSaveWorld#77]
My backups are starting to get rather large, at 200GB> for 7 days worth of backups spaced every 53 minutes. I tried to enable the zip option, but it doesn't do anything and just gets set back to false. My guess is the plugin relies on some stupid Windows syscall rather than the zip utility packaged with just about every mainstream GNU/Linux based OS. Should I just set up cron to do the job instead (and with p7zip)? 

Server:
Linux 5.8.0-50-generic
Ubuntu 20.04 LTS
Paper 1.12.2

**Comment by Shevchik** [comment]
No. First of all, windows doesnt have syscalls for zip. Second, java has built in support for creating zip files. If the config option gets set to false, it means that either your config somehow got malformed after edit before reload, or you didnt reload the settings at all.
